### ISSUE TITLE
`sc-client-api` is not named correctly

### ISSUE THREAD

**Issue by kianenigma** [paritytech/polkadot-sdk#7]
as per our crate naming convention, `-client-` is redundant because it is already implied by `sc`.

Moreover, if I want to nitpick, `sp-consensus-common` should have only been named `sp-consensus` as there is no `/common` in its `Cargo.toml` path.

**Comment by kianenigma** [comment]
@ParityReleases what is the consequence of this, as the crate will be renamed? if it is a hassle, we might as well just ignore it. It does not seem like anyone other than me cares about the name :)

**Comment by kayabaNerve** [comment]
I appreciate clarity/consistency improvements and subscribed to this issue when I noticed it :)

**Comment by Krayt78** [comment]
Hello @kianenigma , i can take a look at this issue. Does it means that we need to first rename that package in his toml and then update every crate that have it as a dependency ?

**Comment by Krayt78** [comment]
Completely forgot about this one, could i be assigned to it pls ?

**Comment by bkchr** [comment]
I don't think that we should do this renaming. While Kian is right, that the naming is a little bit shitty, I think it is not worth the hassle of deleting the crate.
